report/report.go

```diff
@@ -127,7 +127,7 @@ func (report *Report) WriteBackupReportFile(reportFilename string, timestamp str
 	reportInfo := make([]LineInfo, 0)
 	reportInfo = append(reportInfo,
 		LineInfo{Key: "timestamp key:", Value: timestamp},
-		LineInfo{Key: "gpdb version:", Value: report.DatabaseVersion},
+		LineInfo{Key: "database version:", Value: report.DatabaseVersion},
```

Comment:
CI can already run, so it should not be related to the workflow file. It seems that it may be related to this. The specific testing content may need to be contacted with the development team,